### PR TITLE
Expand doc for initial cmds

### PR DESCRIPTION
**Pull request by jbogarthyde**
Add long descriptions and references for the "add", "new" and "update" commands

### CODE REVIEW

File: packages/angular/cli/commands/new.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/new.json",
-  "description": "Creates a new directory and a new Angular app.",
-  "$longDescription": "",
+  "description": "Creates a new workspace and a new default Angular app.",
```
Comment (jenniferfell) [inline packages/angular/cli/commands/new.json]: Let's not use default to mean "initial" or "starter". Thx.

File: packages/angular/cli/commands/new.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Creates and initializes a new Angular app that is the default project for a new workspace.
```
Comment (jenniferfell) [inline packages/angular/cli/commands/new.md]: Love this :-)

File: packages/angular/cli/commands/new.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+* The new workspace folder is given the specified project name. The new app is created at the top level of the workspace folder.
```
Comment (jenniferfell) [inline packages/angular/cli/commands/new.md]: The new app is in the src folder. That's where all the app-specific source code and configuration files live.

File: packages/angular/cli/commands/update.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Before running this command, see the interactive [Angular Update Guide](https://update.angular.io/).
```
Comment (jenniferfell) [inline packages/angular/cli/commands/update.md]: Pefecto

File: packages/angular/cli/commands/update.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/update.json",
-  "description": "Updates your application and its dependencies.",
-  "$longDescription": "",
+  "description": "Updates your application and its dependencies. See \"https://update.angular.io/\",
```
Comment (jenniferfell) [inline packages/angular/cli/commands/update.json]: Pefecto

File: packages/angular/cli/commands/add.md [hunk 6]
```diff
@@ -1,2 +1,3 @@
-Add support for a library in your project, for example adding `@angular/pwa` which would configure
-your project for PWA support.
+Adds the npm package for a published library to your workspace, and configures your default
```
Comment (jenniferfell) [inline packages/angular/cli/commands/add.md]: Tech question: Configures the app set in "defaultProject" or configures the current app project (when executed from within an app project folder)?

File: packages/angular/cli/commands/new.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+
+* The new app's configuration appears in the `projects` section of the `angular.json` workspace configuration file, under its project name.
```
Comment (jenniferfell) [inline packages/angular/cli/commands/new.md]: Please add a note somewhere about prompts. Something like this: The command prompts for information about features to include in the initial app project, such as routing and the CSS file to use.

File: packages/angular/cli/commands/add.md (hunk 6 above)
Comment (jbogarthyde) [reply packages/angular/cli/commands/add.md]: I don't think there is any "current app project" here - it doesn't take a "project" arg, just the package name. I added a note about "defaultProject".

File: packages/angular/cli/commands/new.md (hunk 7 above)
Comment (jbogarthyde) [reply packages/angular/cli/commands/new.md]: Actually I added this before, but messed up the PR and had to start over and recreate the text. Adding it back now.

File: packages/angular/cli/commands/update.json (hunk 5 above)
Comment (filipesilva) [inline packages/angular/cli/commands/update.json]: Heya, this line contains invalid JSON and is breaking the CI.

Comment (alexeagle) [reply packages/angular/cli/commands/update.json]: fixed

**Pull request merged**